form.js

```diff
@@ -0,0 +1,35 @@
+const emailAddress = document.getElementById('myEmail');
+const templateForm = document.getElementById('form');
+const validatorMessage = document.getElementById('validator-message');
+templateForm.addEventListener('submit', (e) => {
+  if (emailAddress.value !== emailAddress.value.toLowerCase()) {
+    e.preventDefault();
+    validatorMessage.textContent = 'Kindly input a valid email address!';
+  } else {
+    validatorMessage.textContent = '';
+  }
+});
+// Start of Local Storage code
+// const emailAddress = document.getElementById('mail');
+const fullName = document.getElementById('name');
+const textMessage = document.getElementById('message');
+// const templateForm = document.getElementById('form');
+templateForm.addEventListener('input', () => {
+  const localStorageObject = {
+    name: fullName.value,
+    email: emailAddress.value,
+    message: textMessage.value,
+  };
+  localStorage.setItem('templateFormData', JSON.stringify(localStorageObject));
+});
+const getData = JSON.parse(localStorage.getItem('templateFormData'));
+function prePopulated() {
+  if (getData) {
+    fullName.value = getData.name;
+    emailAddress.value = getData.email;
+    textMessage.value = getData.message;
+    return getData;
+  }
+  return '';
```

Comment:
- [ ] I checked your Loom video and I understand your worries. However, when I tested the form it was not working as expected. 
![image](https://user-images.githubusercontent.com/43698511/132926971-bf5ef21a-2178-446c-8700-270822358ca2.png)
As you can see in the picture, I checked the localStorage and nothing is stored in it. Here's an example of the expected behavior:

![image](https://user-images.githubusercontent.com/43698511/132927410-21856d07-6935-4748-9905-767c3023ee01.png)
Kindly fix the issue.